src/display/api.js

```diff
@@ -1727,6 +1733,12 @@ var WorkerTransport = (function WorkerTransportClosure() {
           return Promise.reject(new Error('Worker was destroyed'));
         }
 
+        if (typeof document === 'undefined') {
+          // make sure that this code is not executing in node.js, as
```

Comment:
Nit: Please remember to use uppercase letters at the beginning of sentences in comments, i.e. `// Make sure ...` instead.